src/dealDashboard/dealVotes/dealVotes.ts

```diff
@@ -1,36 +1,73 @@
-import { autoinject, bindable } from "aurelia-framework";
+import { autoinject, bindable, computedFrom } from "aurelia-framework";
 import { DealTokenSwap } from "entities/DealTokenSwap";
 import "./dealVotes.scss";
 import { Router } from "aurelia-router";
 import { EthereumService } from "../../services/EthereumService";
-import { AlertService } from "../../services/AlertService";
+import { FundingModal } from "./fundingModal/fundingModal";
+import { DialogService } from "../../services/DialogService";
 
 @autoinject
 export class DealVotes {
   @bindable deal: DealTokenSwap;
 
   fundingModal: HTMLElement;
 
+  everyTextCopy = [
+    {
+      condition: () => !this.deal.majorityHasVoted && !this.deal.representativeVoted(this.ethereumService.defaultAccountAddress),
```

Comment:
Wondering, if we can put these into methods or getter variables.
Some checks are duplicated, and maybe some can be expressed in the terms of others?

Reply:
@hiaux0  do you have some ideas of names for such getters?

Reply:
isRepresentativeEligibleToVote